src/main.zig

```diff
@@ -642,6 +633,7 @@ pub fn main() void { // MARK: main()
 	audio.setMusic("cubyz:cubyz");
 
 	while(c.glfwWindowShouldClose(Window.window) == 0) {
+		openNextStartupWindow();
```

Comment:
Could you explain why you moved this into the loop?
Do we really need to re-check this every frame?
Did you maybe just intend to open both windows at startup?

Reply:
I can't seem to open two windows at once. One will always take priority unless it's in a different frame.

Reply:
Might have to test on another machine, that doesn't consistently have memory errors

Reply:
Well, the last one will be on top, if that's what mean by priority.

Reply:
If I open one window first, it won't show up. If I open it second, it will.

Reply:
Even when in entirely different positions

Reply:
That is weird, the HUD windows are also opened in a single frame, maybe check what the difference between the code that opens the HUD and your code is.

Reply:
Seems it's working now. My best guess is memory errors on my machine.